Controller/SettingsController.php

```diff
@@ -24,7 +24,10 @@ public function modifyAction(Request $request) {
 			'settings' => $allStoredSettings,
 		);
 
-		$form = $this->createForm(new ModifySettingsForm(), $formData);
+		$form = Kernel::VERSION_ID < 20800
+			? $this->createForm(new LegacyModifySettingsForm(), $formData)
+			: $this->get('form.factory')->createNamed('craue_config_modifySettings', 'Craue\ConfigBundle\Form\ModifySettingsForm', $formData);
```

Comment:
@craue composer lets you use different combination of versions when you use the component packages. this is why you should be feature detecting things for the package you use instead of trying to version detect packages based on a different package.

What you are doing here is similar to making user-agent sniffing in the browser and then assuming whether features are supported or no based on that: it is fragile (and not recommended)